src/sample/index.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
```

Comment:
why?

Reply:
was to disable only the log console but I ended up forgetting to change XD

Reply:
fixed